.github/workflows/pytest.yml

```diff
@@ -26,10 +26,14 @@ jobs:
       - name: Unit tests
         run: |
           make test-unit api_version=v3
+          make test-unit api_version=v4
```

Comment:
Maybe you can just use a matrix https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs